services/provider/server/server.go

```diff
@@ -80,17 +82,32 @@ func NewOCSProviderServer(ctx context.Context, namespace string) (*OCSProviderSe
 		return nil, fmt.Errorf("failed to create new StorageClassRequest instance. %v", err)
 	}
 
+	version, err := semver.Make(ocsVersion.Version)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse operator version as semver. %v", err)
+	}
```

Comment:
The value comes from: `github.com/red-hat-storage/ocs-operator/v4/version` so it makes no sense to validate it.

Reply:
- same as https://github.com/red-hat-storage/ocs-operator/pull/2308#discussion_r1423676048
- want to convert version.Version to semver.Version

Reply:
Convert it where you use it and assume error will not occur

Reply:
done.